inc/functions_serverstats.php

```diff
@@ -203,55 +203,8 @@ function build_server_stats($is_install=1, $prev_version='', $current_version=''
 	}
 
 	// Host URL & hostname
+	// Dropped fetching host details since v.1.8.16 as http://www.whoishostingthis.com API seems to be down and this info is not required by MyBB.
 	$info['hosturl'] = $info['hostname'] = "unknown/local";
-	if($_SERVER['HTTP_HOST'] == 'localhost')
```

Comment:
Shouldn't this check be kept? `$info['hosturl']` and `$info['hostname']` are sent, we might need `localhost` informations.

Reply:
Are we at all keeping the track? I thought no. Just left the default values to avoid any unwanted error.
Anyway, added back. No harm with it.

Reply:
No idea. However, as the unknown/local part was there, I assumed it was correct to detect localhost too. If unwanted, we can delete it later on. In the meantime, this PR fixes the hostcheck not working anymore. Good job!